### ISSUE TITLE
router5 1.0.0

### ISSUE THREAD

**Issue by troch** [axefrog/cycle-router5#9]
Hi Nathan,

router5 1.0.0 has been released and I think it will be easier to integrate with cycle-js, if you are still working on it. To summarise what is new in 1.0.0:
- The router itself is now free of side effects, and is simply a box receiving navigation instructions and outputting state objects / error objects. It makes universal javascript straight forward as it is not depending on any web API or global scope.
- For updating URLs and receiving URL updates (popstate events), a plugin named `router5-history` has to be added.
- Plugins, what are they? They are new and can add behaviour and functionality to a router instance. They can register onStart, onStop, onTransitionStart, onTransitionSuccess, onTransitionCancel, onTransitionError methods.

https://github.com/router5/router5/blob/master/CHANGELOG.md
http://router5.github.io/docs/migration.html

**Comment by axefrog** [comment]
Ah nice, thanks for letting me know. I'm currently sidetracked on another project, but when I get back I will be investigating what staltz and the others achieved with their routing solution. At that point I'll judge whether or not to continue with cycle-router5 or to deprecate it.

**Comment by troch** [comment]
OK just let me know, I can definitely help you bringing it forward. If you decide to deprecate it, I might be interested in taking ownership.

**Comment by axefrog** [comment]
@troch I apologise, I never got back to you. I haven't got time to work on this project right now, but I also haven't yet determined whether it's something I want to continue with or not. If you like, I can put you on as a contributor, or you're welcome to submit pull requests if you'd like to push it forward.

**Comment by troch** [comment]
Yes, I am interested. Don't think I'll have the time before the new year though.

**Comment by axefrog** [comment]
No worries, let me know.

**Comment by troch** [comment]
Finally found some time, I have been working on it, developing an example alongside to try to validate it: https://github.com/router5/examples/tree/master/apps/cycle

**Comment by troch** [comment]
@axefrog so I finalised something. Have you determined if it is something you want to keep getting involved with? I would prefer to have control over it: being able to push to it without having to open PRs, and being able to publish new versions. Let me know!

**Comment by axefrog** [comment]
Sorry I totally missed this notification. I'm happy to release it into your custody :)

**Comment by troch** [comment]
Thanks, that's great :)

**Comment by axefrog** [comment]
If you give me a link to the correct repo, I'll add a link on the readme so that yours should be used, rather than mine.
